commit 5dcfd0abde3d8ee0f9ff9787141b72a838c67ea9 (refs/changes/40/5440/3)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2019-02-05T22:15:17+00:00 (8 months ago)
    
    MANTA-4098 Muskie test suite should set PATH when running tests

diff --git a/Makefile b/Makefile
index fe5b943..96f2cf3 100644
--- a/Makefile
+++ b/Makefile
@@ -64,7 +64,8 @@ manta-scripts: deps/manta-scripts/.git
 
 .PHONY: test
 test: $(STAMP_NODE_MODULES)
-	$(NODE) ./node_modules/.bin/nodeunit --reporter=tap \
+	PATH=$(ROOT)/$(NODE_INSTALL)/bin:$(PATH) \
+	    $(NODE) ./node_modules/.bin/nodeunit --reporter=tap \
 	    test/*.test.js test/mpu/*.test.js
 
 .PHONY: release
